test/unit/tools/pep8.rb

```diff
@@ -0,0 +1,22 @@
+module Test
+  module Quality
+    module Tools
+      # Test for the 'bigfiles' tool within the quality gem
+      module Pep8
+        def pep8_expected_args
+          'fake1.py'
+        end
+
+        def expect_pep8_run(quality_checker)
+          @mocks[:quality_checker_class]
+            .expects(:new).with('pep8',
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.